src/core/reducers/addons.js

```diff
@@ -0,0 +1,15 @@
+const initialState = {
+  wat: {slug: 'wat', title: 'Wat is this?'},
+  foo: {slug: 'foo', title: 'The foo add-on'},
+  food: {slug: 'food', title: 'Find food'},
+  bar: {slug: 'bar', title: 'The bar add-on'},
+};
+
+export default function addon(state = initialState, action) {
```

Comment:
Yeah, I found this a little weird at first. I don't totally hate it but not what I went with first.